sketcherMinimizer.h

```diff
@@ -463,10 +445,81 @@ class EXPORT_COORDGEN sketcherMinimizer
     bool getTreatNonterminalBondsToMetalAsZOBs() {return m_treatNonterminalBondsToMetalAsZOBs;}
     void setTreatNonterminalBondsToMetalAsZOBs(bool b) {m_treatNonterminalBondsToMetalAsZOBs = b;}
 
+    /* add bond to m_extrabonds */
+    void addExtraBond(sketcherMinimizerBond* bond);
+
+    /* getters */
+    std::vector<sketcherMinimizerAtom*> getAtoms() { return _atoms; }
+    std::vector<sketcherMinimizerMolecule*> getMolecules() { return _molecules; }
+    std::vector<sketcherMinimizerAtom*> getReferenceAtoms() { return _referenceAtoms; }
+    std::vector<sketcherMinimizerBond*> getReferenceBonds() { return _referenceBonds; }
```

Comment:
These should all return &, and I think that you probably want const/non-const overloads, e.g.:

    std::vector<sketcherMinimizerAtom*>& getAtoms() { return _atoms; }
    const std::vector<sketcherMinimizerAtom*>& getAtoms() const { return _atoms; }

Reply:
+1